README.md

```diff
@@ -46,6 +46,7 @@ Biological image analysis aims to increase our understanding of biology through
 - [OpenCV](https://github.com/opencv/opencv) - Open source computer vision and machine learning software library.
 - [PYME](https://github.com/python-microscopy/python-microscopy) - Open-source application suite for light microscopy acquisition, data storage, visualization, and analysis.
 - [Scikit-image](https://github.com/scikit-image/scikit-image) - Collection of algorithms for image processing.
+- [htsimaging](https://github.com/rraadd88/htsimaging) - Python package for high-throughput single-cell imaging analysis (Python).
```

Comment:
I suggest putting this entry into the "Other" section as it does not seem to be fitting the requirements for general image analysis software.

Reply:
Ok, it fits in the `Yeast imaging` section though, because even though the tool is supposed to be general-purpose, it has only been applied to yeast imaging. 
Would categorizing under `Yeast imaging` be ok?